src/app/components/projects-page/projects-page.component.scss

```diff
@@ -6,6 +6,111 @@
   flex-direction: column;
   justify-content: center;
   align-items: center;
+  margin: 0px 0px 100px 0px;
+}
+
+#featured-projects-container {
+  position: relative;
+  width: 65%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.featured-project {
+  position: relative;
+  height: 40vh;
+  display: flex;
+  margin: 20px 0px;
+  &:hover img {
+    filter: grayscale(0);
+    opacity: 100%;
+  }
+  &:nth-child(odd){
+    flex-direction: row;
+    left: 0;
+    & .featured-project-info {
+      align-items: flex-end;
+    }
+    & .featured-project-images {
+      left: 0px;
+    }
+  }
+  &:nth-child(even){
+    flex-direction: row-reverse;
+    right: 0;
+    & .featured-project-info {
+      align-items: flex-start;
+    }
+    & .featured-project-images {
+      right: 0px;
+    }
+  }
+  justify-content: flex-end;
+  align-items: center;
+
+  & .project-resources {
+    max-height: fit-content;
+    & .project-resource {
+      background: material.get-color-from-palette($blue-palette, 400);
+      margin: 10px 5px;
+    }
+  }
+}
+
+.featured-project-date {
+  margin: 0px;
+}
+.featured-project-title {
+  margin: 0px 0px 5px 0px;
+}
+.featured-project-description-text {
+  margin: 0px;
+}
+
+.featured-project-images {
+  z-index: -1;
+  width: 60%;
+  height: 40vh;
+  position: absolute;
+  background: material.get-color-from-palette($orange-palette, 500);
+  border-radius: 10px;
+  & img {
+    width: 100%;
+    height: 100%;
+    object-fit: cover !important;
+    border-radius: 10px;
+    filter: grayscale(1);
+    opacity: 30%;
+    transition: all 0.5s ease;
+  }
+}
+
+carousel {
+  height: 100% !important;
+  width: 100%;
+  border-radius: 10px;
+  z-index: -1;
+  filter: grayscale(100%);
+  opacity: 40%;
+
+  & img {
```

Comment:
Quite a few & in this file are redundant. A nested selector is implicitly the child of the parent selector

Reply:
Fixed